python/cudf_polars/cudf_polars/experimental/io.py

```diff
@@ -57,10 +57,13 @@ def _(
             for offset in range(0, nrows, length)
         ]
         new_node = Union(ir.schema, None, *slices)
-        return new_node, {slice: PartitionInfo(count=1) for slice in slices} | {
-            new_node: PartitionInfo(count=count)
+        partition_info: MutableMapping[IR, PartitionInfo] = {
```

Comment:
`pylance` wasn't liking the way this was written previously. This should be equivalent.